citar.el

```diff
@@ -345,6 +345,11 @@ always prompt to select."
                (function-item citar-open-notes)
                (function-item citar-open-note))))
 
+(defcustom citar-no-export-fields nil
```

Comment:
Could we rename this to `citar-bibtex-no-export-fields` please? Citar supports multiple bibliography formats and I think we should move towards splitting each of them into its own subpackage, which would be `citar-bibtex` in this case. We can do the refactoring later, but we should keep that in mind for customization options so we don't have to rename and alias/obsolete them in the future.

Reply:
Not an issue, I've made the change as suggested!

Reply:
Can you make sure to also align the commit message(s)?

Reply:
And also change the commit title and message to match the new variable name.

Reply:
Right, forgot about that. Fixed!